packages/components/psammead-story-promo/src/index.stories.jsx

```diff
@@ -21,6 +69,57 @@ const LiveComponent = ({ headline, service, dir }) => (
   </span>
 );
 
+/* eslint-disable-next-line react/prop-types */
```

Comment:
Sorry, hadn't noticed this - but line 72 - 121 here are mostly the same as lines 45 - 94 in `index.test.js`. Maybe they could be pulled into a central place

Reply:
Done :)